frame/ethereum/src/lib.rs

```diff
@@ -481,3 +473,10 @@ impl<T: Config> Module<T> {
 		}
 	}
 }
+
+/// Returns the Ethereum block hash by number.
+impl<T: Config> BlockHashMapping for Module<T> {
```

Comment:
We decided that for convenience is preferable to implement it to an arbitrary struct.

Was discussed here https://github.com/paritytech/frontier/pull/409#discussion_r650910996